Makefile

```diff
@@ -367,8 +369,13 @@ run-server: prepackaged-binaries validate-go-version start-docker ## Starts the
 	@echo Running mattermost for development
 
 	mkdir -p $(BUILD_WEBAPP_DIR)/dist/files
+ifeq ($(RUN_SERVER_IN_BACKGROUND),true)
 	$(GO) run $(GOFLAGS) -ldflags '$(LDFLAGS)' $(PLATFORM_FILES) --disableconfigwatch 2>&1 | \
 	    $(GO) run $(GOFLAGS) -ldflags '$(LDFLAGS)' $(PLATFORM_FILES) logs --logrus &
+else
+	$(GO) run $(GOFLAGS) -ldflags '$(LDFLAGS)' $(PLATFORM_FILES) --disableconfigwatch 2>&1 | \
```

Comment:
This effectively makes `make run-server` a blocking call, which in turn means `make run` will never get to the `run-client` step.

Reply:
Actually, you are right, but we can change the order between run-client and run-server and should work.